src/NuGet.Core/NuGet.Commands/CommandRunners/PackCommandRunner.cs

```diff
@@ -131,10 +131,63 @@ public PackageArchiveReader BuildPackage(PackageBuilder builder, string outputPa
                 PrintVerbose(outputPath, builder);
             }
 
+            if (_packArgs.InstallPackageToV3Feed)
+            {
+                WriteResolvedNuSpecToPackageOutputDirectory(outputPath, builder);
+                WriteSHA512PackageHash(outputPath, builder);
+            }
+
             _packArgs.Logger.LogMinimal(String.Format(CultureInfo.CurrentCulture, Strings.Log_PackageCommandSuccess, outputPath));
```

Comment:
the logging needs to be updated too if you are installing the package

Reply:
Not installing the package, so this is not needed unless I am misunderstanding the request.

Reply:
by installing the package I mean when InstallPackageToV3Feed is set